Tasks/google-play-release/GooglePlay.js

```diff
@@ -27,6 +27,17 @@ if (authType === "JsonFile") {
 }
 
 var apkFile = resolveGlobPath(tl.getPathInput("apkFile", true));
+var apkFileList = [apkFile];
+var additionaApks = tl.getDelimitedInput('additionalApks', '\n');
```

Comment:
Nit: Typo with "additional". Also, it seems like we're using double-quotes throughout this file.